shotover-proxy/src/transforms/protect/crypto.rs

```diff
@@ -39,11 +39,9 @@ pub async fn encrypt(
         kek_id: sym_key.key_id,
     };
 
-    // TODO: investigate using storing as blob instead of text and then use textAsBlob here
-    // We should functionally verify which has better performance before sticking with one.
     Ok(Operand::Const(format!(
-        "'{}'",
-        serde_json::to_string(&protected)?
+        "textAsBlob('{}')",
```

Comment:
We need to remove the textAsBlob otherwise the data would end up stored as a hex string instead of as the raw bytes.

Reply:
The quotes need to go as well. (refer to https://stackoverflow.com/questions/22141620/is-it-possible-in-cassandra-cql3-insert-statement-binary-data )
I would think the int test would fail as is?

Reply:
Oh, it also needs a `0x` prefix as `hex::encode` doesnt include that.